kafka_consumer/README.md

```diff
@@ -1,32 +1,54 @@
-# Kafka_consumer Integration
+# kafka_consumer Integration
 
-## Overview
+# Overview
 
-Get metrics from kafka_consumer service in real time to:
+This Agent check only collects metrics for Kafka broker offset, consumer offset, and consumer lag. If you want to collect many more Kafka metrics, see the kafka check.
 
-* Visualize and monitor kafka_consumer states
-* Be notified about kafka_consumer failovers and events.
+# Installation
 
-## Installation
+The Agent's Kafka consumer check is packaged with the Agent, so simply [install the Agent](https://app.datadoghq.com/account/settings#agent) on your Kafka nodes. If you need the newest version of the check, install the `dd-check-kafka-consumer` package.
 
-Install the `dd-check-kafka_consumer` package manually or with your favorite configuration manager
+# Configuration
 
-## Configuration
+Create a `kafka.yaml` in the Datadog Agent's `conf.d` directory:
 
-Edit the `kafka_consumer.yaml` file to point to your server and port, set the masters to monitor
+```
+init_config:
+#  Customize the ZooKeeper connection timeout here
+#  zk_timeout: 5
+#  Customize the Kafka connection timeout here
+#  kafka_timeout: 5
 
-## Validation
+instances:
+  # - kafka_connect_str: localhost:9092
```

Comment:
I'll link to the `conf.yaml.example` in this case, but for many checks, the example conf _doesn't_ include much better documentation.

I get that you think conf snippets in the README are a waste of time, but for many checks, they're not. Many example confs are too verbose (i.e. poorly written), improperly indented, document deprecated options (i.e. that we shouldn't tempt users to use), and more.

The snippets provide a Quick Start for users who don't want to read the man page for a check they're just starting to tinker with.